monitoring_flows_and_extensions/index.md

```diff
@@ -0,0 +1,335 @@
+---
+title: 'Monitoring Flows & Extensions'
+description: 'We build a logging and monitoring pipeline for Directus applications without using any third-party dependencies.'
+author:
+  name: 'Daniel Band'
+  avatar_file_name: 'daniel.jpg'
+---
+Directus offers a flexible and extensible foundation for creating a wide range of applications. However, to address more complex use cases such as Backend as a Service or internal applications, we will develope a strategy for monitoring the application's state in this article.
+
+What benefits does monitoring give us:
+- Insights into the flow of data
+- Makes a production application debuggable
+- Allows us to listen to events and send out notifications via different channels
+
+## How is event monitoring different from activity logs and revision?
+ In the context of Directus, activity logs allow us to identify changes on a per-collection item basis, providing a trail to retrace the execution of workflows and view the input/output of operations. On the other hand, event logs focus on tracking specific pieces of data. The difference in granularity between activity tracking and event logging makes them complement each other. For instance, in a monitoring pipeline, when a specific event triggers an alert, we can use revisions to trace back the execution flow to analyze the context.
```

Comment:
```suggestion
In the context of Directus, activity logs allow us to identify changes on a per-item basis, providing a trail to retrace the execution of workflows and view the input/output of operations, while event logs focus on tracking specific pieces of data. 

The difference in granularity between activity tracking and event logging makes them complementary. For instance, in a monitoring pipeline, when a specific event triggers an alert, revisions can be used to trace the execution flow and analyze the context.
```

Reply:
This is honestly fantastic context! Breaking it up will help the reader better digest the information. The first paragraph is definitions, and the second contextualises the realtionship.